pyproject.toml

```diff
@@ -44,12 +44,5 @@ graph = [
 ]
 
 [tool.ruff]
-target-version = 'py311'
+target-version = 'py39'
 line-length = 120
-
-[tool.ruff.lint]
-# See complete list : https://docs.astral.sh/ruff/rules/
-select = [
-    "F",     # pyflakes
```

Comment:
Why did you enable the checks then deleted them? (Also in the middle you lost the "E" checks that are enabled in the default ruleset).

Reply:
I did re-enable